ui/apps/platform/src/Containers/MainPage/Navigation/utils.ts

```diff
@@ -0,0 +1,98 @@
+import { IsFeatureFlagEnabled } from 'hooks/useFeatureFlags';
+import { HasReadAccess } from 'hooks/usePermissions';
```

Comment:
Forgive a minor request to move internal `import` statements below external.

Reply:
Good catch - I'll do this right now in the next follow up PR so that the gears of CI continue on this one.